engine/src/main/java/org/destinationsol/game/Hero.java

```diff
@@ -225,4 +228,12 @@ public boolean isDead() {
     public boolean isAlive() {
         return !isDead;
     }
+
+    public boolean isInvincible() {
+        return isInvincible;
+    }
+
+    public void setInvincible(boolean b) {
```

Comment:
Maybe don't use single letter var names in the future ;)
I'll let it slide for now since it's just for an internal setter :+1: